### PR TITLE
Add node-armhf and document template store

### LINKED ISSUE #46: Update README about template store
We need to update the README with some docs / headings about the template store and how it works / how to submit.

### PR DESCRIPTION
**Pull request by martindekov**
Adding node-armhf template since it was missed by me
also adding tempalte store heading and how to apply
your own custom template

Signed-off-by: Martin Dekov (VMware) <mdekov@vmware.com>

Added `node-armhf` since I missed it adding it and in the process saw that. Also added how to add your own template to the store and what the store is.

Closes #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
 * `environment`
 
 Set timeouts etc
+
+# OpenFaaS Template Store
+
+The Template Store are all the templates from OpenFaaS and OpenFaaS-Incubator organizations, along with user-defined templates which were approved by the community.
```
Comment (rgee0) [inline README.md]: 'The Template store _contains_ ....'
'which _have been_ approved by the community.'
Can we link OpenFaaS-Incubator to https://github.com/openfaas-incubator, please?

File: README.md [hunk 2]
```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
...
+
+## How to submit to the Template Store
+
+If you'd like to see your template in the official store follow these steps:
+
+1. Open issue with your template repository
```
Comment (rgee0) [inline README.md]: 'Open an issue to propose the template and include a link to your template repository'
We should create an issue template to help extract the desired information.

File: README.md [hunk 3]
```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
...
+2. Check if the template is approved by the community
```
Comment (rgee0) [inline README.md]: 'Await feedback on, and subsequent approval of, your proposal from the community.'

File: README.md [hunk 4]
```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
 * `environment`
 
 Set timeouts etc
+
+# OpenFaaS Template Store
+
+The Template Store are all the templates from OpenFaaS and OpenFaaS-Incubator organizations, along with user-defined templates which were approved by the community.
+
+## How to submit to the Template Store
+
```
Comment (rgee0) [inline README.md]: Suggest we reiterate the contribution statement somewhere within this section:

`If you'd like to make a submission then raise an issue to propose it. This should follow the [CONTRIBUTION guide](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) for OpenFaaS.`

I think the first sentence is covered by point 1, so its a matter of making the contributing guide link available.

**Comment by martindekov** [comment]
Thank you for the review @rgee0 ! I will amend the changes by EOD

**Comment by rgee0** [comment]
@martindekov are you still aiming to incorporate these changes?  It'd be good to have the readme reflect the progress with the template store.

**Comment by martindekov** [comment]
Sorry @rgee0 missed the comments, I will amend the changes ASAP

File: README.md (hunk 4 above)
Comment (martindekov) [reply README.md]: Done by transforming point one as you proposed ✔️

File: README.md (hunk 3 above)
Comment (martindekov) [reply README.md]: Done ✔️

File: README.md (hunk 2 above)
Comment (martindekov) [reply README.md]: Done ✔️ 
> We should create an issue template to help extract the desired information.

I suppose we should open issue and separate PR for this?

File: README.md (hunk 1 above)
Comment (martindekov) [reply README.md]: Done ✔️

**Comment by martindekov** [comment]
Ok now the PR is updated, thanks again for the feedback @rgee0 and I apologize for the delay 👍

**Comment by martindekov** [comment]
I will actually add some sort of 
### Approved custom templates
csharp added by ...

**Comment by martindekov** [comment]
Also added field showcasing the custom templates that were created by members of the community

**Comment by alexellis** [comment]
@martindekov can you check once more that this is still relevant and required? I ask because I made some edits to the README.md file recently.

**Comment by martindekov** [comment]
Hey  Alex I believe this is needed as we don't have steps for how to submit community templates, the README.md contains information how to add functions, not templates.

**Comment by alexellis** [comment]
Please can you rebase and resubmit?

**Comment by martindekov** [comment]
Of course will do it in a minute

**Pull request closed without merging**